Federation/Federates/CraterTransport/src/CraterTransport.cpp

```diff
@@ -34,7 +34,7 @@ int main(void) {
 
     // Single Payload instance, extends Dynamical Entity and has a number
     // of functions to retrive its values such as acceleration and position
-    HlaPayloadPtr payload = payloadManager->createLocalHlaPayload();
+    HlaPayloadPtr payload = payloadManager->createLocalHlaPayload(L"Payload");
```

Comment:
Creating Dynamic/Static hla entities now requires a name to be passed to the create method. The data type is a wide string which requires the "L" in front of the name